runtime/cere/src/lib.rs

```diff
@@ -913,6 +913,7 @@ impl pallet_contracts::Config for Runtime {
 		pallet_contracts::migration::v13::Migration<Runtime>,
 		pallet_contracts::migration::v14::Migration<Runtime, Balances>,
 		pallet_contracts::migration::v15::Migration<Runtime>,
+		pallet_ddc_nodes::migrations::MigrateToV1<Runtime>,
```

Comment:
This is wrong place. It should be added here https://github.com/Cerebellum-Network/blockchain-node/blob/32dbf724b9a0b182fcb2a373e35644ff355c3dc6/runtime/cere/src/lib.rs#L1418